hangman.py

```diff
@@ -0,0 +1,240 @@
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # wordlist: list of strings
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+
+wordlist = load_words()
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def warnings(value_mistake, warnings_point, guesses_point):
+    """
+    Subtract warnings point and output name of error
+    """
+
+    if warnings_point > 0:
+        warnings_point -= 1
+        print(f"Oops! {value_mistake}")
+        print("You have", warnings_point, "warnings left:", end=" ")
+    else:
+        guesses_point -= 1
+        print("Oops! You have no warnings left, so you lose one guess:", end=' ')
+    return warnings_point, guesses_point
+
+
+def check_star_30(count, secret_word, letters_guessed):
+    pass
+
+
+def letter_in_word(secret_word, letter, letter_guessed, guesses_point, letters_guessed):
+    """
+    Check if letter is word, else subtracts point
+    """
+
+    letters_guessed.add(letter)
+    if letter in secret_word:
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+    else:
+        if letter in LIST_VOWELS:
+            guesses_point -= 2
+        else:
+            guesses_point -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+    return letters_guessed, guesses_point
+
+
+def classic_input(count_star, warnings_point, guesses_point):
+    if count_star != 0:
```

Comment:
Is it possible that you could use this variable only in this function? Maybe then you can remove its usage from other parts of code?